src/antsibull_changelog/yaml.py

```diff
@@ -42,7 +42,9 @@ def load_yaml(path: str) -> Any:
         return yaml.load(stream, Loader=_SafeLoader)
 
 
-def store_yaml(path: str, content: Any, nice: bool = False) -> None:
+def store_yaml(
+    path: str, content: Any, nice: bool = False, sort_keys: bool = False
```

Comment:
The default value for `sort_keys` is `True`, not `False`:
```suggestion
    path: str, content: Any, nice: bool = False, sort_keys: bool = True
```
Also, let's make this a keyword-only arg:
```suggestion
    path: str, content: Any, /, nice: bool = False, sort_keys: bool = True
```
(We should have already done that with `nice`, but it's too late for that now.)